src/Supplier.php

```diff
@@ -96,6 +97,13 @@ public function cleanDBonPurge()
         Rule::cleanForItemAction($this, 'suppliers_id%');
     }
 
+    public function getCloneRelations(): array
+    {
+        return [
+            Contact_Supplier::class,
+        ];
+    }
+
```

Comment:
Were linked contacts cloned in 9.5?
More generally, cloning a supplier doesn't seem very relevant to me. Can you find out exactly when this was removed? I'm wondering if the fact that it was clonable in 9.5 was actually a mistake that we have since fixed.

Reply:
A lot of things in 9.5 were clonable but only because it was enabled at the `CommonDBTM` level which was obviously problematic. I reworked the feature for 10.0 and made cloning less global and with more consideration for if an itemtype should be cloned (Plugin and Config were technically clonable in 9.5) at all and how it should be cloned. Suppliers were not one of the itemtypes I considered during the rework.

As far as I can tell, the original cloning implementation didn't have any relations cloned for Suppliers.